pages/account/statistics.js

```diff
@@ -0,0 +1,83 @@
+import { authOptions } from "../api/auth/[...nextauth]";
+import { unstable_getServerSession } from "next-auth/next";
+
+import Page from "../../components/Page";
+import PageHead from "../../components/PageHead";
+
+export async function getServerSideProps(context) {
+  const session = await unstable_getServerSession(
+    context.req,
+    context.res,
+    authOptions
+  );
+
+  if (!session) {
+    return {
+      redirect: {
+        destination: "/",
+        permanent: false,
+      },
+    };
+  }
+
+  let data = {};
+  try {
+    const res = await fetch(
+      `${process.env.NEXT_PUBLIC_BASE_URL}/api/account/statistics`,
```

Comment:
@Cahllagerfeld this is the request that fails, the api rejects with no session found - but visit the API path in the browser is fine

Reply:
Resolved by passing 

```js
headers: {
   cookie: context.req.headers.cookie || "",
}
```